imports/plugins/included/payments-paystack/client/settings/containers/paystackSettingsFormContainer.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+import { TranslationProvider } from "/imports/plugins/core/ui/client/providers";
+import { Reaction, i18next } from "/client/api";
+import { PaystackSettingsForm } from "../components";
+
+class PaystackSettingsFormContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      sKey: "278302390293",
+      pKey: "278302390293"
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.saveUpdate = this.saveUpdate.bind(this);
+  }
+
+  handleChange(e) {
```

Comment:
If you write this function as 
```js
handleChange = e => {...}
```
you won't need to declare the binding in the constructor. Same thing applies to `handleSubmit` and `saveUpdate`